closure/testing/test/BUILD

```diff
@@ -83,6 +83,22 @@ closure_js_test(
     ],
 )
 
+closure_js_library(
+    name = "goog_testSuite_lib",
+    srcs = ["goog_testSuite.js"],
+)
```

Comment:
Seems like you don't need the library to reproduce your issue.

Reply:
You're right!  I've removed the library.